sql/test_data.sql

```diff
@@ -83,8 +83,17 @@ INSERT INTO position_holders(pos_id, user_id) VALUES
     (4, 2),
     (5, 4);
 
+
 INSERT INTO position_relations(pos_id_from, pos_id_to) VALUES
     (4, 5);
 
 INSERT INTO rooms(room_id, location, title, description) VALUES
     (1, 'SAC 23', 'ASCIT Screening Room', 'A room for watching DVDs and videos');
+
+INSERT INTO permissions(permission_id, permission_name) VALUES
+    (1, 'Ruddock full member'),
+    (2, 'IHC member');
```

Comment:
Also  minor nit: 
It'll be better for readability later if you gave these better names. Right now they sounds more like position names. Perhaps you can do like "Rudocck Website Access" or "Can view Rotation Violations" to be more clear as to what they are